src/Neo.VM/ExecutionEngine.cs

```diff
@@ -1681,6 +1681,7 @@ public StackItem Pop()
         /// </summary>
         protected virtual void PostExecuteInstruction(Instruction instruction)
         {
+            if (ReferenceCounter.Count < Limits.MaxStackSize) return;
             if (ReferenceCounter.CheckZeroReferred() > Limits.MaxStackSize)
                 throw new InvalidOperationException($"MaxStackSize exceed: {ReferenceCounter.Count}");
```

Comment:
Does we have a ut that test this exception?

Reply:
not a thing for this pr. Its jus keeps conflict and conflic and conflic, please stop considering none pr issues.

Reply:
If you want to trust that this single line doesn't break anything, you must ensure that it works, so yes, this is a thing for this pr.

Reply:
It should break things. It a limit of the vm.

Reply:
No, VM has its limitatoins. 2G at most actally.

Reply:
I dont mean in a bad way.